tests/js/spec/views/settings/incidentRules/ruleForm.spec.jsx

```diff
@@ -0,0 +1,93 @@
+import {mount} from 'enzyme';
+import React from 'react';
+
+import {initializeOrg} from 'app-test/helpers/initializeOrg';
+import {selectByLabel} from 'app-test/helpers/select';
+import {RuleFormContainer} from 'app/views/settings/incidentRules/ruleForm';
+
+describe('Incident Rules Form', function() {
+  const {organization, project, routerContext} = initializeOrg();
+  const rule = TestStubs.IncidentRule();
+  let createRule;
+  const createWrapper = props =>
+    mount(
+      <RuleFormContainer
+        organization={organization}
+        orgId={organization.slug}
+        projects={[project, TestStubs.Project({slug: 'project-2', id: '3'})]}
+        {...props}
+      />,
+      routerContext
+    );
+  beforeEach(function() {
+    MockApiClient.clearMockResponses();
+    createRule = MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/alert-rules/',
+      method: 'POST',
+    });
+  });
+
+  describe('Creating a new rule', function() {
+    /**
+     * Note this isn't necessarily the desired behavior, as it is just documenting the behavior
```

Comment:
Why isn't this desired. Seems like preserving the excluded projects between toggle states is helpful.

Reply:
Yeah I agree, especially in the edit case where it saves on change. I just haven't fully made up my mind on it - I could also see the case where people toggle it and the label change itself isn't eye-catching enough and they don't realize it's a different field.